tests/avocado/test_vnc.py

```diff
@@ -0,0 +1,43 @@
+from avocado_qemu import test
+
+
+class Vnc(test.QemuTest):
+    """
+    :avocado: enable
+    :avocado: tags=vnc
```

Comment:
Do we have a tag for quick tests? Or tests that don't really execute qemu (or execute only in stopped state?)? If so this test is a good candidate (otherwise we might want to include it).